src/main/java/org/jabref/logic/cleanup/RenamePdfCleanup.java

```diff
@@ -33,18 +33,22 @@
     private final BibDatabaseContext databaseContext;
     private final boolean onlyRelativePaths;
     private final String fileNamePattern;
-    private final LayoutFormatterPreferences layoutPreferences;
+    // FIXME: remove unused class member later (S.G.):
```

Comment:
Please remove the commented-out code here.

Reply:
Fixed.